packages/indexer/lib/index.js

```diff
@@ -88,9 +102,32 @@ class Indexer {
     }, this.livenessCheckIntervalSeconds * 1000);
   }
 
-  collector({ lock = null, type = null, argsLen = -1, data = "0x" } = {}) {
+  collector({ lock = null, type = null, argsLen = -1, data = "any" } = {}) {
     return new CellCollector(this, { lock, type, argsLen, data });
   }
+
+  subscribe({
+    lock = null,
+    type = null,
+    argsLen = -1,
+    data = "any",
+    fromBlock = null,
+  } = {}) {
```

Comment:
I would suggest we do a little more here: here we are accepting all fields included in QueryOptions. When a developer is passing in a value that subscribe won't use, such as `toBlock` or `skip`, we shall generate a log message saying the passed field is ignored.

Reply:
updated.